account_banking_pain_base/models/account_payment_order.py

```diff
@@ -50,31 +31,14 @@ class AccountPaymentOrder(models.Model):
         "false, the bank statement will display one debit line per wire "
         "transfer of the SEPA XML file.")
 
-    @api.multi
-    @api.depends(
-        'company_partner_bank_id.acc_type',
-        'payment_line_ids.currency_id',
-        'payment_line_ids.partner_bank_id.acc_type')
-    def compute_sepa(self):
-        eur = self.env.ref('base.EUR')
-        for order in self:
-            sepa = True
-            if order.company_partner_bank_id.acc_type != 'iban':
-                sepa = False
-            for pline in order.payment_line_ids:
-                if pline.currency_id != eur:
-                    sepa = False
-                    break
-                if pline.partner_bank_id.acc_type != 'iban':
-                    sepa = False
-                    break
-            sepa = order.compute_sepa_final_hook(sepa)
-            self.sepa = sepa
-
-    @api.multi
-    def compute_sepa_final_hook(self, sepa):
-        self.ensure_one()
-        return sepa
```

Comment:
I personally don't use this hook but I suppose it was there for a reason. Not sure how to preserve it though.

Reply:
I think it's not needed anymore because the way the sepa field is computed has changed.

It was used in l10n_ch (https://github.com/OCA/l10n-switzerland/blob/0e006e5bcbff4c64bfc5fee8dd28123d0b139d31/l10n_ch_pain_base/models/account_payment_order.py#L13) to deactivate the sepa, but as I change the way the sepa field is now computed.

I don't know if it's used somewhere else. Do you prefer if I let this hook just in case?